specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -0,0 +1,3344 @@
+                            "$ref": "#/definitions/billingRoleDefinition"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/projects/{projectName}/providers/Microsoft.Billing/billingRoleDefinitions/{billingRoleDefinitionId}": {
+            "get": {
+                "tags": [
+                    "billingRoleDefinition"
+                ],
+                "x-ms-examples": {
+                    "ProjectRoleDefinition": {
+                        "$ref": "./examples/ProjectRoleDefinition.json"
+                    }
+                },
+                "operationId": "ProjectbillingRoleDefinition_Get",
+                "description": "Gets the role definition for a role",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/billingAccountNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/projectNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/billingRoleDefinitionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK. The request has succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/billingRoleDefinition"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/providers/Microsoft.Billing/billingRoleDefinitions/{billingRoleDefinitionId}": {
+            "get": {
+                "tags": [
+                    "billingRoleDefinition"
+                ],
+                "x-ms-examples": {
+                    "BillingProfileRoleDefinition": {
+                        "$ref": "./examples/BillingProfileRoleDefinition.json"
+                    }
+                },
+                "operationId": "BillingProfilebillingRoleDefinition_Get",
+                "description": "Gets the role definition for a role",
+                "parameters": [
+             
```

Comment:
The only outstanding issue here was creation of roleassignment using POST. The service team has agreed to explore this after the preview release.